### DISCUSSION TITLE
Will TS ever be natively supported?

### DISCUSSION THREAD

**Discussion by evilC** [mifi/lossless-cut#1743] [General]
Here is an example of one of the files that I typically have to work with. It's already been cut with LosslessCut (So things seem to work fine), it's just the low quality, lack of audio etc. Wondering if there are any plans to make it fully supported?
[knife edge gap to dive.zip](https://github.com/mifi/lossless-cut/files/12910288/knife.edge.gap.to.dive.zip)

**Comment by mifi** [comment]
planned to improve non-native formats, but no timeline unfortunately. see #88

**Comment by evilC** [comment]
I just had a quick test of [this build](https://github.com/mifi/lossless-cut/actions/runs/8957389735).  
I can confirm that I now have audio.  
I can also now drag the timeline bar and scrub through with the mouse, which I don't remember being able to do before.
However, I did still see the "File not natively supported" popup.

**Reply by mifi** [reply]
MPEGTS files are not natively supported, however in the upcoming version (now nightly), the playback of these files has been improved, including audio support. You can see the "FFmpeg-assisted playback" showing up in the top left corner when you are playing such a file.
